tests/serverpod_test_client/test/copy_with_mutable_test.dart

```diff
@@ -96,6 +96,21 @@ void main() {
       );
```

Comment:
There is also the file `tests/serverpod_test_server/test/generated_methods/copy_with_mutable_test.dart` (test server vs. test client).


Since the models are not shared, we'll have to copy the same test over there as well.

Reply:
Added new tests for `copyWith` on `BigInt`.